src/entities/Votes/types.ts

```diff
@@ -40,3 +40,5 @@ export type VoteCount = {
 export type Voter = {
   address: string
 } & VoteCount
+
+export type VotesForProposals = Record<string, Record<string, Vote>>
```

Comment:
(Not for this PR but maybe we could be explicit here on what is exactly `string` on both cases) 
E.g: `export type VotesForProposals = Record<SnapshotProposal['id'], Record<Voter['address'], Vote>>`

Reply:
😍